front/src/services/api/additionalServicesFetch.js

```diff
@@ -1,29 +1,47 @@
-var additionalServices = [
-    {
-        name: 'начос',
-        price: 3
-    },
-    {
-        name: 'cola',
-        price: 2
-    }
-]
-
-export default additionalServices;
-
-const delay = (ms) =>
-    new Promise(resolve => setTimeout(resolve, ms));
-
 export const addAdditionalService = (item) =>
-    delay(500)
-        .then(() => {
-            additionalServices.push(item);
+    fetch(
+        "./api/additionalservices",
```

Comment:
It is much worth to have baseUrl setting instead of '.'. There is absolutely no information about where your api is hosted